torch_geometric/transforms/to_sparse_tensor.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         attr: Optional[str] = 'edge_weight',
         remove_edge_index: bool = True,
         fill_cache: bool = True,
-        layout: Optional[int] = None,
+        layout: Optional[torch.layout] = None,
```

Comment:
```suggestion
        layout: Optional[int] = None,
```
This needs to be `int` at the moment :(